### PR TITLE
Add Nova Scotia shields

### PR DESCRIPTION
**Pull request by claysmalley**
Adds shields for Nova Scotia. This introduces a new shield for 100-series "Arterial" routes, and factors the US route shield into a generic reusable shield, as this design has been reused in a few parts of the Western Hemisphere.

Screenshot from 2022-04-09 17-44-49
Screenshot from 2022-04-09 17-45-14

Nova Scotia's one- and two-digit routes are Trunk highways (`network=CA:NS:T`). Shields for these highways have a suspiciously similar design to US routes. I'm not quite sure of the history behind this, or who did what first, but they're almost identical, except for the white background where the US shield has a black background.

Nova_Scotia_1

Nova Scotia's 100-series is Arterial highways (`network=CA:NS:H`), which are limited-access highways similar to Ontario's 400-series, Quebec's Autoroutes, or the Interstate system in the US. The design introduced in this PR removes all the red details on the yellow shield-within-a-shield, compresses the flag vertically and adjusts the colours to match the existing palette.

Nova_Scotia_101

Nova Scotia's 200- and 300-series are Collector highways (`network=CA:NS:R`), linking rural areas to population centres. These have a simple brown rectangular shield with white text, almost identical to National Forest Roads in the US (#246).

Nova_Scotia_Route_201

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -90,6 +90,22 @@ export function loadShields(shieldImages) {
...
+    shieldImages.shield40_us_us_2,
+    shieldImages.shield40_us_us_3,
+  ];
+  var padding_us_us = {
+    left: 2,
+    right: 2,
+    top: 4,
+    bottom: 5,
+  };
+  var usMUTCDNationalShield = {
```
Comment (1ec5) [inline style/js/shield_defs.js:103]: If you’re looking for a more convenient variable name, I think it’s called a “badge” shield.

**Review by 1ec5 — APPROVED** [review]
The arterial shield is beautiful – gives me hope that someday we can find a way to squeeze a little more detail into shields like South Carolina’s without sacrificing legibility.

File: style/js/shield_defs.js (hunk 1 above)
Comment (claysmalley) [reply style/js/shield_defs.js:103]: Good thinking; I'll go ahead and rename that.

**Comment by claysmalley** [comment]
Once we have a complete inventory of shields, we can spend some time refining the earliest ones. I have improvements in mind for the Carolinas 😄

**Comment by ZeLonewolf** [comment]
> The arterial shield is beautiful – gives me hope that someday we can find a way to squeeze a little more detail into shields like South Carolina’s without sacrificing legibility.

Legibility of the _text_... you certainly can't tell that there's a red dragon in there ;)

**Review by ZeLonewolf — APPROVED** [review]
Looks good!

**Pull request merged**